egs/wsj_mix/asr1/RESULTS.md

```diff
@@ -0,0 +1,23 @@
+| :-----: | :-: | :--: | :-: | :-: | :-: | :-: |
+| exp/tr_pytorch_vggblstmp_sde1_rece2_subsample1_2_2_1_1_unit1024_proj1024_d1_unit300_location_spafalse_aconvc10_aconvf100_mtlalpha0.2_adadelta_sampprob0.0_bs10_mli1000_mlo150_lsmunigram0.05_delta/decode_cv_beam30_emodel.acc.best_p0.0_len0.0-0.0_ctcw0.3_rnnlm1.0_1layer_unit1000_sgd_bs300_word65000 | 503 | 85377 | 6581 | 4890 | 3982 | 15.96 |
+| exp/tr_pytorch_vggblstmp_sde1_rece2_subsample1_2_2_1_1_unit1024_proj1024_d1_unit300_location_spafalse_aconvc10_aconvf100_mtlalpha0.2_adadelta_sampprob0.0_bs10_mli1000_mlo150_lsmunigram0.05_delta/decode_tt_beam30_emodel.acc.best_p0.0_len0.0-0.0_ctcw0.3_rnnlm1.0_1layer_unit1000_sgd_bs300_word65000 | 333 | 60849 | 3537 | 2695 | 1920 | 12.15 |
+| exp/tr_pytorch_vggblstmp_sde1_rece2_subsample1_2_2_1_1_unit1024_proj1024_d1_unit300_location_spatrue_aconvc10_aconvf100_mtlalpha0.2_adadelta_sampprob0.0_bs10_mli800_mlo150_lsmunigram0.05_delta/decode_cv_beam30_emodel.acc.best_p0.0_len0.0-0.0_ctcw0.3_rnnlm1.0_1layer_unit1000_sgd_bs300_word65000 | 503 | 85388 | 5927 | 5533 | 2875 | 14.80 |
+| exp/tr_pytorch_vggblstmp_sde1_rece2_subsample1_2_2_1_1_unit1024_proj1024_d1_unit300_location_spatrue_aconvc10_aconvf100_mtlalpha0.2_adadelta_sampprob0.0_bs10_mli800_mlo150_lsmunigram0.05_delta/decode_tt_beam30_emodel.acc.best_p0.0_len0.0-0.0_ctcw0.3_rnnlm1.0_1layer_unit1000_sgd_bs300_word65000 | 333 | 61630 | 3176 | 2275 | 1842 | 10.87 |
+## WER
+| dataset | Snt | Corr | Sub | Del | Ins | Err |
+| :-----: | :-: | :--: | :-: | :-: | :-: | :-: |
+| exp/tr_pytorch_vggblstmp_sde1_rece2_subsample1_2_2_1_1_unit1024_proj1024_d1_unit300_location_spafalse_aconvc10_aconvf100_mtlalpha0.2_adadelta_sampprob0.0_bs10_mli1000_mlo150_lsmunigram0.05_delta/decode_cv_beam30_emodel.acc.best_p0.0_len0.0-0.0_ctcw0.3_rnnlm1.0_1layer_unit1000_sgd_bs300_word65000 | 503 | 12691 | 3169 | 566 | 651 | 26.70 |
+| exp/tr_pytorch_vggblstmp_sde1_rece2_subsample1_2_2_1_1_unit1024_proj1024_d1_unit300_location_spafalse_aconvc10_aconvf100_mtlalpha0.2_adadelta_sampprob0.0_bs10_mli1000_mlo150_lsmunigram0.05_delta/decode_tt_beam30_emodel.acc.best_p0.0_len0.0-0.0_ctcw0.3_rnnlm1.0_1layer_unit1000_sgd_bs300_word65000 | 333 | 9350 | 1677 | 291 | 308 | 20.11 |
+| exp/tr_pytorch_vggblstmp_sde1_rece2_subsample1_2_2_1_1_unit1024_proj1024_d1_unit300_location_spatrue_aconvc10_aconvf100_mtlalpha0.2_adadelta_sampprob0.0_bs10_mli800_mlo150_lsmunigram0.05_delta/decode_cv_beam30_emodel.acc.best_p0.0_len0.0-0.0_ctcw0.3_rnnlm1.0_1layer_unit1000_sgd_bs300_word65000 | 503 | 12817 | 2921 | 688 | 475 | 24.86 |
+| exp/tr_pytorch_vggblstmp_sde1_rece2_subsample1_2_2_1_1_unit1024_proj1024_d1_unit300_location_spatrue_aconvc10_aconvf100_mtlalpha0.2_adadelta_sampprob0.0_bs10_mli800_mlo150_lsmunigram0.05_delta/decode_tt_beam30_emodel.acc.best_p0.0_len0.0-0.0_ctcw0.3_rnnlm1.0_1layer_unit1000_sgd_bs300_word65000 | 333 | 9557 | 1501 | 260 | 308 | 18.28 |
+
+The mixture scheme is in the local/wsj_mix_scheme.tar.gz
```

Comment:
Did you include this in `run.sh`? I'm a bit confused.

Reply:
Do you mean the data preparation of wsj_mix? I did not include the mixture generation, which is the same scripts except to prepare the single-speaker source files according to the schemes in local/wsj_mix_scheme.tar.gz for now. But the text generation for two speakers and other files needed are included in `local/wsj_2mix_data_prep.sh`.

Reply:
Then, where is `local/wsj_mix_scheme.tar.gz` used?

Reply:
Previously, this script was mainly focused on wsj0_2mix which was made officially public by MERL. I did not include the mixture scheme of wsj_2mix for simiplicity. Maybe I can also provide the full process of preparing wsj_2mix this time.

Reply:
> Previously, this script was mainly focused on wsj0_2mix which was made officially public by MERL. I did not include the mixture scheme of wsj_2mix for simiplicity. 

I got it. For now, please add more comments to clarify it there. 

> Maybe I can also provide the full process of preparing wsj_2mix this time.

I think this is important. Please make another PR to include wsj_2mix.